ford_fusion_sehybrid_2019/docker-compose.yml

```diff
@@ -129,8 +121,6 @@ services:
     network_mode: host
     volumes_from:
       - container:carma-config:ro
-    environment:
-      - CYCLONEDDS_URI=${CYCLONEDDS_URI}
```

Comment:
Can we remove the `/opt/carma/vehicle/calibration` volume for velodyne_lidar_driver as well?